spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -0,0 +1,3 @@
+# Preview all emails at http://localhost:3000/rails/mailers/user_mailer
```

Comment:
drop comments

Reply:
Sorry, now that I think of it, this comment is actually helpful. 
Where does the `rails` part in the path come from?